lib/drivers/abbott/abbottFreeStyleNeo.js

```diff
@@ -0,0 +1,415 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2020, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import _ from 'lodash';
+import sundial from 'sundial';
+
+import annotate from '../../eventAnnotations';
+import common from '../../commonFunctions';
+import TZOUtil from '../../TimezoneOffsetUtil';
+import FreeStyleProtocol from './freeStyleLibreProtocol';
+import { KETONE_VALUE_FACTOR, KETONE_HI } from './freeStyleLibreConstants';
+
+const isBrowser = typeof window !== 'undefined';
+// eslint-disable-next-line no-console
+const debug = isBrowser ? require('bows')('PrecisionNeoDriver') : console.log;
+
+const RECORD_TYPE = {
+  GLUCOSE: 7,
+  KETONE: 9,
+  INSULIN: 10,
+  BASAL_TITRATION: 11,
+  TIME_CHANGE: 6,
+};
+
+class PrecisionNeo {
+  constructor(cfg, protocol) {
+    this.cfg = cfg;
+    this.hidDevice = this.cfg.deviceComms;
+    this.protocol = protocol;
+  }
+
+  async commandResponse(cmd) {
+    return new Promise((resolve, reject) => {
+      this.protocol.requestTextResponse(cmd, (result) => {
+        resolve(result);
+      },
+      (err) => {
+        debug('requestTextResponse Error:', err);
+        reject(err);
+      });
+    });
+  }
+
+  async getSerialNumber() {
+    const serialNumber = await this.commandResponse('$serlnum?');
```

Comment:
```suggestion
    return this.commandResponse('$serlnum?');
```

And delete the next line.

Reply:
:+1: